src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -372,6 +377,32 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_round() {
+        let tests = vec![
+            (ScalarFuncSig::RoundInt, Datum::I64(1), Datum::I64(1)),
+            (
+                ScalarFuncSig::RoundInt,
+                Datum::I64(i64::MAX),
+                Datum::I64(i64::MAX),
+            ),
+            (
+                ScalarFuncSig::RoundInt,
+                Datum::I64(i64::MIN),
+                Datum::I64(i64::MIN),
+            ),
+        ];
+
+        let mut ctx = EvalContext::new(Arc::new(EvalConfig::new().set_ignore_truncate(true)));
```

Comment:
Why not `EvalConfig::default_for_test()` here?

Reply:
Why not add `EvalContext::default_for_text` ?